esque/cli/commands.py

```diff
@@ -1118,3 +1120,134 @@ def ping(state: State, times: int, wait: int):
+                          (See "Available Handlers" below)
+    <serializer scheme> - Define data serialization format for key and value in message (e.g json or avro).\b
+                          If no value serializer scheme defined key scheme will be used for both.
+                          (See "Available Serializers" below)
+    <host> and <path>   - Depends on handler and defines the exact source or target of the data pipeline.
+                          (See "Available Handlers" below)
+    <param>             - Define parameters for handler and/or serializer. Use the following prefixes to define
+                          which component a parameter should be used for:
+                            k__  use as key serializer parameter
+                            v__  use as value serializer parameter
+                            kv__ use as key and value serializer parameter
+                            h__  use as handler parameter
+                          E.g. `k__indent=1` will pass the parameter `indent` with a value of `1` to the key serializer.
+                          Use `esque urlencode some&string:with;special_characters` to make values that would interfere
+                          with uri parsing safe to use as query parameters.
+                          (For supported parameters, see "Available Handlers" and "Available Serializers" below)
+
+    Available Handlers:
+
+    \b
+    - kafka
+      <host> -> Used for esque context. If empty, use active context.
+      <path> -> Used as topic name.
+      Supported Parameters:
+        * consumer_group_id
+          Use this consumer group id, when reading from a topic. (Default is "esque-client")
+
+    \b
+    - pipe
+      <host> -> Valid values are "stdin", "stdout" or "stderr".
+      <path> -> No meaning, should stay empty.
+      Supported Parameters:
+        * key_encoding
+          How to embed/extract the message key and value into/from the json objects that are written/read to/from the
+          pipe. Valid values are "base64", "utf-8" and "hex". Use "utf-8" if you know the values are strings and don't
+          mess up the output. Otherwise "base64" is a safer choice. (Default is "utf-8")
+        * value_encoding
+          Same as `key_encoding` but for the values. (Default is "utf-8")
+        * skip_marker
+          Use any value (preferably 1) if you don't want any marker lines to be written between json objects.
+          The marker line "__esque_msg_marker__\\n" is required for esque to be able to **read** from a pipe.
+          Therefore, you can skip the markers in the output when you're not intending to read the piped data back in.
+          You'll even **need** to skip the marker if you want to further process the messages with tools like `jq`.
+
+    \b
+    Available Serializers:
+    - json
+      Supported Parameters:
+        * indent
+          When serializing data into json, use this many spaces to indent (pretty format) the output.
```

Comment:
By default the serializer uses no line breaks or indentation and puts everything into one line.